modules/apps/fragment/fragment-web/src/main/java/com/liferay/fragment/web/internal/portlet/action/EditFragmentEntryMVCActionCommand.java

```diff
@@ -51,18 +54,45 @@
 		"javax.portlet.name=" + FragmentPortletKeys.FRAGMENT,
 		"mvc.command.name=/fragment/edit_fragment_entry"
 	},
-	service = MVCActionCommand.class
+	service = {AopService.class, MVCActionCommand.class}
```

Comment:
Should only have AopService here otherwise 2 MVCActionCommands will be registered, 1 proxy and 1 non-proxy of this.